### ISSUE TITLE
ElvUI skin optimization: Can't see what category is selected

### ISSUE THREAD

**Issue by Taizunz** [TheKrowi/Krowi_AchievementFilter#122]
With the ElvUI Achievement skin enabled it's impossible to see what category has been selected in the achievement frame.

Example images without and with ElvUI skin - notice the visual difference on the selection of the Stormheim category:
![image](https://user-images.githubusercontent.com/15648000/234196126-498470e1-7ab5-4cc6-95f3-d5746de7a5d4.png)
![image](https://user-images.githubusercontent.com/15648000/234196138-11bd863c-48e0-474f-bc25-dd862a4627a4.png)

**Comment by Taizunz** [comment]
Yes. With just ElvUI enabled the selected category is highlighted in white:
![image](https://user-images.githubusercontent.com/15648000/234248805-12dc0451-fbc3-4d28-bfec-4739fbca221a.png)

**Comment by TheKrowi** [comment]
Something I forgot in that case.

**Comment by TheKrowi** [comment]
Will be fixed when 10.1 releases.
![image](https://user-images.githubusercontent.com/22818581/234371970-15e73022-8c02-4cb9-b4f8-be89d3746638.png)
